### ISSUE TITLE
Import the configuration

### ISSUE THREAD

**Issue by lilililizheng** [sysrepo/sysrepo#2936]
hi Can sysrepo import the following xml configuration?
![image](https://user-images.githubusercontent.com/106649263/196078402-4e7c82ee-8bc5-4cd5-b68c-04d2bc85e76b.png)
I hope to complete the operation of the picture and delete this ip address under the interface. But I execute sysrepocfg --import=interfaces.xml and the result is as follows：
![image](https://user-images.githubusercontent.com/106649263/196078662-3b63334c-3e2f-42a8-bd79-e12155c840cb.png)

**Comment by lilililizheng** [comment]
Can only netconf do this?

**Comment by michalvasko** [comment]
> Can only netconf do this?

Yes, but you can do that with *sysrepo* as well except you must use the `edit` operation instead of `import`.

**Comment by lilililizheng** [comment]
thanks，I found that out. Now I want to specify sysrepocfg -- how should rpc be used? I executed sysrepo --rpc= interfaces.xml.netpeer2-server and received the message, but it did not work and the address was not removed.
![image](https://user-images.githubusercontent.com/106649263/196119565-d3b9b457-3a1c-434b-a1db-a3012830e425.png)

**Comment by lilililizheng** [comment]
![image](https://user-images.githubusercontent.com/106649263/196121284-d74ca71c-67fd-44c9-a77f-04611ac529f2.png)
I execute sysrepocfg --rpc=vim.Nothing happens, and as sysrepocfg --help explains, the running library data should be returned to me and output to stdout

**Comment by michalvasko** [comment]
No, I meant just use the original XML with an `edit` operation and not `import`. Meaning `sysrepocfg --edit=<xml-file>`.

**Comment by lilililizheng** [comment]
Sorry, in case I wasn't clear, the sysrepocfg --edit=<xml-file> command works.I was wodnering how sysrepo -rpc changed to use, As I described above, it doesn't seem to react at all

**Comment by lilililizheng** [comment]
![image](https://user-images.githubusercontent.com/106649263/196126082-e0d293de-1de1-40e3-b1b1-30358b08e746.png)
As described by --rpc, I used the change command correctly without reporting any errors, but I didn't get the expected result

**Comment by michalvasko** [comment]
I would look at the server log, there may be some problem such as NACM blocking the access. It should work normally.

**Comment by lilililizheng** [comment]
If you block access, you should get an error, but netpeer2-server does not get any error.

**Comment by lilililizheng** [comment]
![image](https://user-images.githubusercontent.com/106649263/196131547-ccf2c276-b4d5-40ea-a86f-89485ce9c8bc.png)

**Comment by michalvasko** [comment]
I see. So you are saying that the edit config succeeds but the value is not deleted? That seems weird and if so, then I would need to reproduce it, what *sysrepo* version are you using?

**Comment by lilililizheng** [comment]
Of course, not all edit-config fails. When I use edit-config --target running --config=interfaces.xml in netopeer2-cli, the result is successful. This only fails at sysrepocfg --rpc=interfaces.xml

**Comment by michalvasko** [comment]
Okay, I have looked at the code and it seems this is intentional. *netopeer2* ignores all non-NETCONF RPCs, which I believe makes sense except it could print some message and probably fail, so I have improved that.

**Comment by lilililizheng** [comment]
Yes, I just found this out, all the RPCS defined using sysrepocfg --rpc and netconf will fail, but the custom ones, such as the oven defined in the sysrepo example, will succeed

**Comment by lilililizheng** [comment]
So when using sysrepocfg --rpc, should you prompt the user to only operate on custom RPCS?

**Comment by michalvasko** [comment]
> should you prompt the user to only operate on custom RPCS?

`sysrepocfg` will not prompt anything, the error is clear now. But, in general, it does not make much sense and could even be a security problem.
